pkgs/tools/misc/seaborn-data/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib, newScope, fetchFromGitHub, unzip, stdenvNoCC }:
+let
+  base = {
+    version = "master";
```

Comment:
I missed something there. Version shouldn't be master, it should be "unstable-20xx-xx-xx" where the xx are replaced with the date of the commit you are choosing so 2023-09-16 for example.

Reply:
Oh, good to know, fixed!